packages/db/drizzle/0021_idle_follow_up_seed.sql

```
@@ -0,0 +1,51 @@
+        "instanceId": "{{payload.instanceId}}",
+        "to": "{{payload.chatId}}",
+        "contentTemplate": "{{followUpReply}}"
+      }
+    }
+  ]'::jsonb,
+  true,
+  100,
+  NOW(),
+  NOW()
+WHERE NOT EXISTS (
+  SELECT 1 FROM "automations" WHERE "name" = 'Idle chat follow-up'
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)